config/302-pac-configmap.yaml

```diff
@@ -33,6 +33,17 @@ data:
   # PipelineRun on the Tekton dashboard
   tekton-dashboard-url: ""
 
+  # Enable or disable the feature to show a log snippet of the failed task when there is
+  # an error in a Pipeline
+  #
+  # It will show the last 3 lines of the first container of the first task
+  # that has error in the pipeline.
+  #
+  # you may want to disable this if you think your pipeline may leak some value
+  error-log-snippet: "true"
```

Comment:
by default true?

Reply:
i guess if pm says otherwise then we will disable it 🤷🏻